pkg/dao/templates_test.go

```diff
@@ -221,3 +222,52 @@ func (s *TemplateSuite) TestListFilterSearch() {
 	assert.Len(s.T(), responses.Data, 1)
 	assert.Equal(s.T(), found[0].Name, responses.Data[0].Name)
 }
+
+func (s *TemplateSuite) TestDelete() {
```

Comment:
It could just be included in this test, but can you also add a test for using the "Delete" method (i.e. not "SoftDelete").

Reply:
Testing change looks good!

Why do we need to check if the record exists in the delete method? I ask because by default deleting will return "record not found" as an error

Reply:
without that check i was seeing some weird behavior in the test, where even with a bad org ID the record was deleted without error. maybe i was doing something wrong in the test though, i'll remove the check again and see if i can just fix the test

Reply:
changed the test to check for an invalid uuid instead org id. still not sure why it would succeed with an invalid org id though